lsl/common/stations.py

```diff
@@ -652,6 +626,17 @@ def __sub__(self, std):
                 out = (self.x-std, self.y-std, self.z-std)
                 
         return out
+        
+    @property
+    def earth_location(self):
+        if self._parent is None:
+            raise RuntimeError("Cannot determine stands location without a reference point")
+            
+        center = self._parent.earth_location
+        aa = AltAz(CartesianRepresentation(self.y*astrounits.m, self.x*astrounits.m, self.z*astrounits.m),
+                   location=center)
+        aa = aa.transform_to(ITRS())
+        return EarthLocation.from_geocentric(aa.x, aa.y, aa.z)
```

Comment:
This is one area where we need to check to make sure that we are getting the correct answer.

Reply:
There was problem, it's fixed now (and tested for as well).